src/panel/login_masq.js

```diff
@@ -0,0 +1,66 @@
+import LoginMasqPanelView from '../views/login_masq.dot'
+import Panel from "../libs/panel";
+import Store from "../adapters/store"
+import Error from '../adapters/error'
+import nconf from "../../local_modules/nconf_getter";
+let moduleConfig = nconf.get().store
+
+
+export default class LoginMasqPanel {
+  constructor() {
+    this.panel = new Panel(this, LoginMasqPanelView)
+    if(moduleConfig.name === 'masq') {
+      this.store = new Store()
+      this.isActive = false
+      this.loggingIn = false
+      this.username = null
+      this.profileImage = null
+      listen('login_panel__show', () => {
+        this.panel.animate(.25, '.login_masq_panel', {top : '100px'})
+      })
+
+      this.isLoggedIn = false
+      this.store.isLoggedIn().then(async (b) => {
```

Comment:
can you move this to an async function so you can clean async syntax

Reply:
moved to an async function called in the constructor without await